src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -261,7 +261,7 @@ class Program
         #endregion C# 7.3
 
         [Fact]
-        public async Task UpgradeAllProjectsToCSharp7()
+        public async Task UpgradeAllProjects()
```

Comment:
❗️ The old test should be preserved in its current form. The new test should be added as `UpgradeAllProjectsToCSharp8`. (Applies to all tests modified in this file.)

Reply:
The latest is now changed, so IIUC, this will always upgrade to 8.

---
In reply to: [185126820](https://github.com/dotnet/roslyn/pull/26405#discussion_r185126820) [](ancestors = 185126820)